web/src/lib/components/forms/library-user-picker-form.svelte

```diff
@@ -37,6 +39,7 @@
     <p class="p-5 text-sm">NOTE: This cannot be changed later!</p>
```

Comment:
Assuming https://github.com/immich-app/immich/pull/9188 is merged, then you will be able to change the readonly status, so we might need to clarify what "This" is referring to in this message